packages/input-option/src/InputOption/themes.ts

```diff
@@ -0,0 +1,169 @@
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+
+const State = {
+  Default: 'default',
+  Hover: 'hover',
+  Focus: 'focus',
+  Disabled: 'disabled',
+  Checked: 'checked',
+  Destructive: 'destructive',
+} as const;
```

Comment:
I don't think these states are mutually exclusive. An item can be `checked` + `hovered` at the same time. A user can `focus` on a `destructive` item.

Reply:
this doesn't mean that theyre mutually exclusive (if you look at how hovered, focus styles are applied)